src/condor_ce_host_network_check

```diff
@@ -136,16 +236,47 @@ def iface_matches(network_iface, pattern):
     return False
 
 def is_local_addr(ipv4_addr):
+    """Return if an IPv4 address is on a local network, i.e. one of
+    - 192.168.0.0/16
+    - 172.16.0.0/16
```

Comment:
Might as well note that 172.16.0.0 really should be a /12, but that's kinda hard to do in python...